ChangeLog

```diff
@@ -1,4 +1,138 @@
-### 3.9.1dev <- NOTE: the release version number will be 3.9.2 ###
+### 3.9.1dev <- NOTE: the release version number will be 3.10.0 ###
+
+- Refactor: use `size_t` for vector and array indexes that must not be negative (#2893).
+  (contributed by @pljones)
+
+- RPC: Added new command line switch (`--jsonrpcbindip`) to specify the bind address for the RPC server (#2917).
+  (contributed by @Rob-NY)
+
+- RPC: jamulusserver/getClients method expanded to include all elements of a user profile as well as the number of active client connections (#2918).
+  (contributed by @Rob-NY)
+
+- CONDENSE with https://github.com/jamulussoftware/jamulus/pull/2833 (#3008).
+  (contributed by @ann0see)
+
+- RPC: Rename `directoryServer` to `directory` in `jamulusserver/getServerProfile` response (#3048).
+  (contributed by @pljones)
+
+- Removed docs/RELATED-PROJECTS.md from repository. The up to date version is now on jamulus.io (#3091).
+  (contributed by @mcfnord)
+- GUI: Make checkboxes self-labelling (#2996).
```

Comment:
The blank line has vanished.

Reply:
Strange. Maybe a bug in the script?

Reply:
The content is from a comment, not the PR body. So this might be the problem.

Reply:
Fixed. The comment didn't have a new line after the sentence ended.